fusor-ember-cli/app/mirage/config.js

```diff
@@ -103,11 +103,21 @@ export default function() {
   });
 
   this.get('/fusor/api/openstack/deployments/:id/flavors', function(db, request) {
-    return db.flavors;
+    // NOTE root node is flavor and not flavors
+    return {flavor: db.flavors};
```

Comment:
@jmontleon, @jwmatthews, maybe the flavor API should be updated to return 'flavors' to be consistent. This messed me up for a while until I realized it returned singular `flavor`